docker-compose.yml

```diff
@@ -28,7 +28,21 @@ services:
       POSTGRES_PASSWORD: helpwave
```

Comment:
Are these overwritten when they exist in the environment or just default values? I'm not sure about docker-composes behavior.

Reply:
I guess the order would be the following:
Dockerfile
docker-compose
.env
manual variable